handler-host.go

```diff
@@ -66,12 +66,19 @@ type HostHandler struct {
 	// will attempt to read this directory for any existing files and
 	// send them to the host.
 	CacheFileLocation string
-	db *sql.DB
+	db                *sql.DB
 
 	// List of filenames/Ids that are currently being sent to the server,
 	// so they do not get sent again.
 	currentlySending   []string
 	currentlySendingMu sync.RWMutex
+
+	// Channel to stop processing
+	die           chan bool
+	stopHeartbeat chan bool
```

Comment:
Do we need so many? Is it risky to reuse one?

Reply:
bad habit.